core/src/main/java/org/apache/accumulo/core/util/Halt.java

```diff
@@ -49,15 +49,16 @@ public void run() {
   }
 
   public static void halt(final int status, Runnable runnable) {
+
     try {
       // give ourselves a little time to try and do something
-      new Daemon() {
+      Threads.createThread("Halt Thread", new Runnable() {
         @Override
         public void run() {
           sleepUninterruptibly(100, TimeUnit.MILLISECONDS);
           Runtime.getRuntime().halt(status);
         }
-      }.start();
+      }).start();
```

Comment:
With a lambda, this would be slightly shorter:

```java
      Threads.createThread("Halt Thread", () -> {
        sleepUninterruptibly(100, TimeUnit.MILLISECONDS);
        Runtime.getRuntime().halt(status);
      }).start();
```

Reply:
Resolved in latest commit